docs/guides/install/index.html

```diff
@@ -3,215 +3,248 @@
+          </body>
+        </html>
+      {% endcode %}
+    </section>
+
+    <section><!-- Polyfills -->
+      <h2 class="hxSectionTitle">Polyfills</h2>
+      <p>
+        HelixUI requires two polyfills to ensure maximum compatibility.
+      </p>
+    </section>
+
+    <section><!-- polyfill.io -->
+      <h2 class="hxSectionTitle">polyfill.io</h2>
+      <p>
+        The first polyfill is an online service that uses browser detection to
+        serve the smallest set of polyfills needed to accommodate missing functionality
+        (<a href="https://polyfill.io">polyfill.io</a>).
+      </p>
+      <p>
+        Load the script before HelixUI and any application logic.
+      </p>
+      {% code 'html' %}
+        <body>
+          ...
+
+          <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+
+          <!-- HelixUI custom element definitions go here -->
+
+          <!-- Application logic goes here -->
+        </body>
+      {% endcode %}
+    </section>
+
+    <section><!-- webcomponentsjs -->
+      <h2 class="hxSectionTitle">webcomponentsjs</h2>
+      <section>
+        <p>
+          The second polyfill adds support for web component technologies, using
+          two scripts.
+        </p>
+      </section>
+
+      <section>
+        <h3 class="hxSubSectionTitle">Web Components Polyfill Loader</h4>
+        <div class="hxAlert">
+          <div class="hxAlert__icon">
+            <hx-icon type="info-circle"></hx-icon>
+          </div>
+          <div class="hxAlert__text">
+            <span class="hxAlert__status">NOTE</span>
+            Currently, the loader only supports being loaded in the <code>&lt;head&gt;</code>
+            of an HTML document.
+          </div>
+        </div>
+        <br />
+
+        <p>
+          The loader performs feature detection on the current browser to determine
+          so that it can dynamically inject the smallest polyfill needed to patch
+          missing functionality to the end of the <code>&lt;head&gt;</code> of your
+          application.
+        </p>
+        {% code 'html' %}
+          <head>
+            ...
+
+            <script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+            <!-- polyfills injected here (if needed) -->
+          </head>
+        {% endcode %}
+        <br />
+
+        <p>
+          Once the polyfills are loaded, a "WebComponentsReady" event will fire
+          on the document to announce availability of Web Component functionality.
+        </p>
+      </section>
+
+      <section>
+        <h3 class="hxSubSectionTitle">Custom Elements ES5 Adapter</h4>
+        <p>
+          The Custom Elements v1 spec requires that constructor classes be defined
+          using ES6 class syntax.  This is problematic, because HelixUI is compiled
+          to ES5 syntax in order to support legacy browsers like Internet Explorer
+          that do not support ES6.
```

Comment:
Consider rewording to "The ES6 class syntax requirement can be problematic. To support legacy browsers such as Internet Explorer that do not support ES6, the HelixUI is compiled to ES5 syntax."